docs/setup_installation/gcp/getting_started.md

```diff
@@ -32,12 +32,24 @@ In [managed.hopsworks.ai](https://managed.hopsworks.ai/) click on *Connect to GC
   </figure>
 </p>
 
-## Step 2: Creating and configuring a storage
+## Step 2: Creating storage
 
-The Hopsworks clusters deployed by [managed.hopsworks.ai](https://managed.hopsworks.ai/) store their data in a bucket in your GCP account. To enable this you need to create a bucket and to create a service account with permissions to access the storage.
+The Hopsworks clusters deployed by [managed.hopsworks.ai](https://managed.hopsworks.ai/) store their data in a bucket in your GCP account. This bucket need to be created before the cluster creation.
 
-### Step 2.1: Creating a custom role for accessing storage 
+Execute the following gsutil command to create a bucket. Replace all occurrences \[PROJECT_ID\] with your GCP project id and \[BUCKET_NAME\] by the name you want to give to your bucket:
+
+```
+gsutil mb -p [PROJECT_ID] gs://[BUCKET_NAME]
+```
+
+!!! note 
+    The Hopsworks cluster created by [managed.hopsworks.ai](https://managed.hopsworks.ai/) must be in the same region as the bucket. The above command will create the bucket in the US so in the following steps, you must deploy your cluster in a US region. If you want to deploy your cluster in another part of the word us the *-l* option of *gsutil mb*. For more detail about creating buckets with gsutil see the [documentation](https://cloud.google.com/storage/docs/creating-buckets)
```

Comment:
For more details about creating buckets with gsutil, see the [google documentation]